GraphicEditor/Resources/Styles/ButtonStyles.xaml

```diff
@@ -0,0 +1,83 @@
+                    xmlns:local="clr-namespace:GraphicEditor.Resources.Styles">
+    
+    <Style TargetType="Button" x:Key="TriangleButton">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="Button">
+                    <Grid>
+                        <Polygon Fill="Coral"
+                                 Points="-1,-2 193,0 193,180"
+                                 Stroke="Black"
+                                 StrokeThickness="2"/>
+                        <ContentPresenter HorizontalAlignment="Center" VerticalAlignment="Center">
+                        </ContentPresenter>
+                    </Grid>
+                </ControlTemplate>
+            </Setter.Value>
+        </Setter>
+        <Setter Property="Cursor" Value="Hand"/>
+    </Style>
+
+    <Style TargetType="Button" x:Key="UButton">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="Button">
+                    <Grid>
+                        <Polygon Fill="Purple"
+                                 Points="0,0 60,0 60,80 100,80 100,0 160,0 160,145 0,145 "
+                                 Stroke="Black"
+                                 StrokeThickness="3"/>
+                        <ContentPresenter HorizontalAlignment="Center" VerticalAlignment="Center">
+                        </ContentPresenter>
+                    </Grid>
+                </ControlTemplate>
+
+            </Setter.Value>
+        </Setter>
+        <Setter Property="Cursor" Value="Hand"/>
+    </Style>
+
+    <Style TargetType="Button" x:Key="OctagonButton">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="Button">
+                    <Grid>
+                        <Polygon Fill="Yellow"
+                                 Points="30,2 97,2 127,64 96,124 30,124 -2,63"
+                                 Stroke="Black"
+                                 StrokeThickness="3"/>
+                        <ContentPresenter HorizontalAlignment="Center" VerticalAlignment="Center">
+                        </ContentPresenter>
+                    </Grid>
+                </ControlTemplate>
+
+            </Setter.Value>
+        </Setter>
+        <Setter Property="Cursor" Value="Hand"/>
+    </Style>
+
+    <Style TargetType="Button" x:Key="DermatologyButton">
+        <Setter Property="Template">
+            <Setter.Value>
+                <ControlTemplate TargetType="Button">
+                    <Grid>
+                        <Rectangle Fill="Chocolate"
+                                   Width="150"
+                                   Height="250"
+                                   Stroke="Black"
+                                   StrokeThickness="3"/>
+                        <!--<Image Source="pack://application:,,,/Resources/Images/parkDermatology.png" 
```

Comment:
Do we need this?

Reply:
No. It was just useful in one moment in order to set a good position of an object on the map.